quilt/test/test_push.py

```diff
@@ -21,14 +21,12 @@ def find_object_hashes(contents):
     Iterator that returns hashes of all of the tables.
     """
     for key, obj in contents[CHILDREN_KEY].items():
-        print("KEY=%s, OBJ=%s" % (key, obj))
-        if key == TYPE_KEY:
-            continue
         obj_type = NodeType(obj[TYPE_KEY])
         if obj_type is NodeType.TABLE or obj_type is NodeType.FILE:
-            yield from obj["hashes"]
+            hash_it = iter(obj['hashes'])
+            yield next(hash_it)
```

Comment:
I'm pretty sure you need a loop here and below? Doesn't this just return the first item?